intellij/src/saros/intellij/preferences/colors/SarosColorsPage.java

```diff
@@ -0,0 +1,173 @@
+package saros.intellij.preferences.colors;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.intellij.openapi.editor.colors.TextAttributesKey;
+import com.intellij.openapi.fileTypes.PlainSyntaxHighlighter;
+import com.intellij.openapi.fileTypes.SyntaxHighlighter;
+import com.intellij.openapi.options.colors.AttributesDescriptor;
+import com.intellij.openapi.options.colors.ColorDescriptor;
+import com.intellij.openapi.options.colors.ColorSettingsPage;
+import java.text.MessageFormat;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+import javax.swing.Icon;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import saros.intellij.editor.colorstorage.ColorManager;
+import saros.intellij.editor.colorstorage.ColorManager.IdentifiableColorKeys;
+import saros.intellij.ui.Messages;
+import saros.intellij.ui.util.IconManager;
+
+/** IntelliJ color scheme preferences for Saros specific colors. */
+public class SarosColorsPage implements ColorSettingsPage {
```

Comment:
I just want to remember you that there is an option to select your color for a session. Maybe use a better fitting name.

Reply:
How about `LocalSarosAnnotationColorsPage`?

I am assuming that you are hinting at the future need for a settings page where you can set your preferred color (that other see your annotations in), which could be confused with this page. If this is not the case, please explain what you mean.

Reply:
You guessed it right.

Reply:
Good. Any issues with the suggested name?

Reply:
Nope. Can you please post a picture what this page looks like ? Maybe we can introduce that too in Saros/E.

Reply:
There already is a picture of this page in the initial PR #482 (I also linked it below). The page did not change much except that I changed the text to state which annotation for which "user" (color id) is shown.

![Preferences](https://user-images.githubusercontent.com/1101857/54837251-351f7380-4cc6-11e9-9c8c-8cf6199bde50.jpg)

We could add a separate page for this, but these settings are already available in Saros/E under "General" > "Editors" > "Text Editors" > "Annotations", which should have multiple entries starting with "DPP".

I just noticed that we also have to rename these. I will open an issue for this.

Reply:
Nice. The Eclipse stuff is pretty poor and cumbersome to use and currently changing the values is not the best idea as we are replacing them on every IDE start with the default values. 😈 

![image](https://user-images.githubusercontent.com/41400679/62724167-a4e29080-ba12-11e9-9cbd-460bd77a9e53.png)